docs/api_reference2.rst

```diff
@@ -0,0 +1,7 @@
+Surfactant
+=================
+
+.. automodule:: Surfactant
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
This file can be deleted -- the info on it will appear on the api_reference page.